masappcli/masappcli.py

```diff
@@ -351,7 +375,11 @@ def store_scan_summary_from_scan_id(self, scan_id):
 
         """
 
-        user_scan_summary = self.auth_user.get_scan_summary(self.scan_info["wg"], scan_id)
+        if self.scan_info["wg"] == None:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [comparison to None should be 'if cond is None:'](https://app.codacy.com/app/alopezna5/mASAPP_CI/pullRequest?prid=4655639)